### PR TITLE
taskotron to use mtf from copr and install python2-avocado-plugins-va…

### PR DESCRIPTION
**Pull request by bgoncalv**
…rianter-yaml-to-mux plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Install prerequisite RPMs if necessary:
 * pdc-client
 * python2-aexpect - dependency for python-avocado
 * python2-avocado - avocado testing framework
+* python2-avocado-plugins-varianter-yaml-to-mux (Required for avocado-51.0 or later)
```
Comment (contyk) [inline README.md]: The surrounding lines all use a dash rather than parens.

Comment (bgoncalv) [reply README.md]: updated

**Comment by contyk** [comment]
+1

File: runtask.yml [hunk 2]
```diff
@@ -12,13 +12,19 @@ environment:
         - pdc-client
         - python2-aexpect
         - python2-avocado
+        - python2-avocado-plugins-varianter-yaml-to-mux
```
Comment (mmathesius) [inline runtask.yml]: This plugin is currently available only in the Fedora update-testing repos, so the package needs to be installed via a separate shell action by running  `dnf -y --enablerepo=updates-testing install python2-avocado-plugins-varianter-yaml-to-mux`.

Comment (mmathesius) [reply runtask.yml]: Be sure to install the corresponding `python2-avocado` from the updates-testing repo as well.

**Review by mmathesius — APPROVED** [review]
Looks good to me. 👍

**Review by igor-raits — COMMENTED** [review]
not sure if running dnf operations during test is good idea...

**Comment by mmathesius** [comment]
Merging to 'develop' branch now. Please follow-up to make sure it's working properly from taskotron since avocado 51.0 will be appear in to the stable repos later this week.

**Pull request merged**